### PR TITLE
fix : Turn on the microphone with the current device id

### PR DESCRIPTION
**Pull request by zjzhang-cn**
#### Description
根据DeviceID打开mic，修复多个mic设备的不能选择的bug。
Open microphone with DeviceID. fix the bug that multiple mic devices cannot be selected.
#### Reference issue
Fixes #...

### CODE REVIEW

File: pkg/driver/microphone/microphone.go [hunk 1]
```diff
@@ -111,6 +111,8 @@ func (m *microphone) AudioRecord(inputProp prop.Media) (audio.Reader, error) {
 	config.PerformanceProfile = malgo.LowLatency
 	config.Capture.Channels = uint32(inputProp.ChannelCount)
 	config.SampleRate = uint32(inputProp.SampleRate)
+	//FIXME: Turn on the microphone with the current device id
```
Comment (at-wat) [inline pkg/driver/microphone/microphone.go:114]: Is it not fixed yet and to be fixed in future? (FIXME?)

Comment (zjzhang-cn) [reply pkg/driver/microphone/microphone.go]: It has been modified，

File: pkg/driver/microphone/microphone.go [hunk 2]
```diff
@@ -111,6 +111,8 @@ func (m *microphone) AudioRecord(inputProp prop.Media) (audio.Reader, error) {
 	config.PerformanceProfile = malgo.LowLatency
 	config.Capture.Channels = uint32(inputProp.ChannelCount)
 	config.SampleRate = uint32(inputProp.SampleRate)
+	//FIX: Turn on the microphone with the current device id
```
Comment (at-wat) [inline pkg/driver/microphone/microphone.go:114]: As it seems already fixed by this PR, removing `FIX:` would be less confusing.

Maybe
```suggestion
	// Select the microphone with the current device id
```
?

**Review by at-wat — APPROVED** [review]
The fix looks good to me

**Pull request merged**